paasta_tools/cli/cmds/status.py

```diff
@@ -534,6 +573,13 @@ def report_status_for_cluster(
                 for line in status.rstrip().split('\n'):
                     output.append('    %s' % line)
 
+    for name, instance in instance_whitelist.items():
```

Comment:
this will display status for every action in every job for a given service. we should instead have single status item per job consisting of some info about every action of that job preferably in topological order - tron api provides all the data necessary for that.